src/main/java/duke/Ui.java

```diff
@@ -5,24 +5,41 @@
 
 import java.util.Scanner;
 
+/**
+ * Ui class to manage user input and Duke output
+ */
 public class Ui {
     private static final String space = "     ";
     private static final String divider = "    -------------------------------------------->>>>>\n";
     public Scanner sc;
 
+    /**
+     * Ui constructor
+     */
     public Ui() {
         this.sc = new Scanner(System.in);
     }
 
+    /**
+     * Prints welcome message
+     */
     public void printWelcomeText() {
         System.out.println(divider + space + "Konichiwa~ Watashi wa Duke desu! \n" + space
                 + "What can I do for you today?\n" + divider);
     }
```

Comment:
I found you code easy to understand and easy to read. The quality of code was good and you adhered to coding standards. The names of methods were appropriate as well. Keep it up